src/NBomber/Contracts.fs

```diff
@@ -64,11 +65,19 @@ type Statistics = {
 
 type IConnectionPool<'TConnection> = interface end
 
+/// Data provider
+[<Interface>]
```

Comment:
why [<Interface>]?

can it be just shortened?

type IFeed<'T> =
    /// Feed name, which is also the key of step data dictionary
    abstract Name: string
    /// Gets data for the next step
    abstract Next: unit -> Dict<string,'T>

Reply:
@vilinski  can you please for now use 'T instead of 'a just to keep the same style.
I am ok to change it later on 'a but for all cases at once. For now, it just adds inconsistency in the style. I do understand that 'a is kind of standard in F# world and we should start using it.

Reply:
`[<Interface>]` ? dunno, removed. thought otherwise C# would see it as an abstract class instead

adjusted `'a` -> `'T`

Reply:
@vilinski it's opposite and exectly for this you have [<AbstractClass>]

Reply:
✓